### ISSUE TITLE
Review / update About page

### ISSUE THREAD

**Issue by kateefly** [DLFMetadataAssessment/Sandbox#95]
Needs to be reviewed; breadcrumbs implemented?

**Comment by GentrySteven** [comment]
Tried playing around with breadcrumbs on the "About" page--mixed results. Whenever I try to implement breadcrumbs, the link redirects me to a non-existent "home" page in the entries folder. Thoughts on how to fix so that the breadcrumb can redirect to the index.md file? Or is this even necessary?

Other changes currently made on my fork only (attached here as a .txt for your review):
- References to the "Metadata Assessment Toolkit" have been removed--it seems odd to only refer to it in a few places, but not elsewhere on the site. Could also cause confusion, since we appear to be introducing both ourselves and this "toolkit", even if that is what our online resource really is.
- Terminology has been standardized--the working group is now called the "DLF Metadata Assessment Working Group" (instead of other variants, like "DLF AIG Metadata Assessment Working Group"). I will also plan on updating the wiki and google form to reflect this terminology, assuming folks are alright with it.
- I broke up the "About Page" into paragraphs that I feel are more digestible. I also tried to specifically highlight our mission. Let me know if this is correct.

Also, should we remove the section concerning our previous efforts? The DLF Metadata Assessment wiki page (https://wiki.diglib.org/Assessment:Metadata) already lists previous efforts, and so it seems redundant to repeat this kind of information (I'm also looking for a way to make the content of the About page fit on the screen).

General thoughts?
[about.txt](https://github.com/DLFMetadataAssessment/Sandbox/files/3464601/about.txt)
